docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/entity-name-is-not-unique.md

```diff
@@ -66,7 +60,7 @@ NotUnique:TransferName Transfer object name "CustomProductData" has to have proj
 
 ## Transfer property name is not unique
 
-Transfer property names must be unique. Check the examples below to learn how to fix the error.
+Transfer property names must be unique, when you extend transfer that exists on the vendor level. Below you can find the code examples that show how to solve the described problem.
```

Comment:
What is a vendor level? I've only heard about core level before.

Reply:
It is the same 
replaced to `core level`